chowda/routers/sony_ci.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter
+from metaflow.integrations import ArgoEvent
+from pydantic import BaseModel
+from typing import List
+
+router = APIRouter(prefix='/sony_ci')
+
+
+class SyncResult(BaseModel):
+    inserted: int
+    deleted: int
+    errors: List[str]
+
+
+@router.post('/sync', tags=['sync'])
+async def sony_ci_sync() -> SyncResult:
+    ArgoEvent('sync').publish()
+    # TODO: use correct values for inserted, deleted, and errors
+    return SyncResult(inserted=0, deleted=0, errors=[])
```

Comment:
I think the role of the API here is just to emit the event, not to watch the process, since the sync will take several minutes to complete.

It should return:
- `200` immediately on successful `publish()`
- `500` if there was an error publishing the event.

Also, we should store the sync event in the database, so we can update the dashboard. Is this a CLAMS event?